scripts/run-integration-tests.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-if [[ "$1" == "chrome" && "$2" == "cypress/e2e/device.cy.ts" ]]; then
-  npm run test:integration -- --spec 'cypress/e2e/device.cy.ts' --browser chrome
+if [[ -n "$2" ]]; then
+  npm run test:integration -- --spec "$2" --browser chrome
```

Comment:
Let's think of a way to be able to use this script feature with browsers other than chrome. Right now, we won't be able to pass both firefox and a spec right?

Reply:
It also appears that we don't report failures in this case? Should add something like `&& echo pass || bash scripts/report-failure.sh`